.env

```diff
@@ -74,14 +74,14 @@ CKANEXT__SPATIAL__SEARCH_BACKEND=solr-bbox
 # Customize map widget
 CKANEXT__SPATIAL__COMMON_MAP__TYPE=custom
 
-# CKANEXT__SPATIAL__COMMON_MAP__CUSTOM_URL=https://tile.openstreetmap.org/{z}/{x}/{y}.png
-# CKANEXT__SPATIAL__COMMON_MAP__ATTRIBUTION=<a href=https://openstreetmap.org/>OpenStreetMap</a> contributors
+CKANEXT__SPATIAL__COMMON_MAP__CUSTOM_URL=https://tile.openstreetmap.org/{z}/{x}/{y}.png
```

Comment:
Do you mind adding comments for each of these sets of vars about what the intended use of each is, @FuhuXia ?

Reply:
comments added.